config/authorization/config.lisp

```diff
@@ -0,0 +1,51 @@
+;;;;;;;;;;;;;;;;;;;
+;;; delta messenger
+; (in-package :delta-messenger)
+
+; (add-delta-logger)
+; (add-delta-messenger "http://delta-notifier/")
```

Comment:
You can add `delta-notifier` to the stack as well, currently only with a rule to notify mu-cl-resource

Reply:
Had been added by @x-m-el , I still added a dash, for consistency in service naming between compose file and sparql-parser config.